src/helpers/walletKitUtil.ts

```diff
@@ -28,15 +28,12 @@ import { getDappMetadataFromEvent } from "hooks/useDappMetadata";
 import { TFunction } from "i18next";
 import { ToastOptions } from "providers/ToastProvider";
 import { analytics } from "services/analytics";
+import { submitTx } from "services/stellar";
 
 /** Supported Stellar RPC methods for WalletKit */
 const stellarNamespaceMethods = [
   StellarRpcMethods.SIGN_XDR,
   StellarRpcMethods.SIGN_AND_SUBMIT_XDR,
-
-  // This is actually a Ethereum RPC method, but it's being used by Allbridge
-  // so let's just recognize it as a valid method so we don't reject their connection
-  StellarRpcMethods.PERSONAL_SIGN,
```

Comment:
cleaning up this method as Allbridge fixed stuff on their side so we don't need it anymore